virttest/libvirt_vm.py

```diff
@@ -489,6 +489,40 @@ def add_numa(vcpus, max_mem, numa_nodes):
             cmd += " %s" % cells
             return cmd.strip(",")
 
+        def pin_numa(help_text, host_numa_node_list):
+            """
+            Method to pin guest numa with host numa
+            :param help_text: virt-install help message to check the option
+            :param host_numa_node_list: list of online host numa nodes
+
+            :return: parameter to pin host and guest numa with virt-install
+            """
+            if not has_option(help_text, "numatune"):
+                return ""
+            cmd = " --numatune"
+            numa_pin_mode = params.get("numa_pin_mode", "strict")
+            # If user gives specific host numa nodes to pin by comma separated
+            # string pin_to_host_numa_node = "0,1,2", check if the numa
+            # node is in online numa list and use.
+            host_numa = str(params.get("pin_to_host_numa_node"))
```

Comment:
We can have this as an option for user to provide specific numa nodes, else we will pin all the host online nodes with memory.

The reason for doing a str type cast to consider node 0, is user gives `pin_to_host_numa_node = 0` it will be int and `if host_numa` condition will not succeed.